service/models.py

```diff
@@ -105,14 +111,16 @@ def deserialize(self, data):
         return self
 
     @classmethod
-    def init_db(cls, app):
+    def init_db(cls, app: Flask):
         """ Initializes the database session """
         logger.info("Initializing database")
-        cls.app = app
```

Comment:
This would have been part of the template right? Do we need to remove this?

Reply:
Im not sure about this I thought it might reinitialize the app or something. so I removed it. I can add it back again as it was part of the template

Reply:
Done